test/http-data-source.test.ts

```diff
@@ -1336,12 +1337,15 @@ test('Should only cache GET successful responses with the correct status code',
   t.is(cacheMap.size, 0)
 })
 
-test('Global maxAge should be used when no maxAge was set or similar.', async (t) => {
+test.serial('Global maxAge should be used when no maxAge was set or similar.', async (t) => {
```

Comment:
using `serial()` here because otherwise FakeTimers mess up the timeouts in concurrent tests